docs/adr/adr-0021-from-rem-to-em.md

```diff
@@ -0,0 +1,111 @@
+However, this setting may be completely ignored by some browsers when the website font sizes have been set using fixed units like `px`. 
+This is the case with Chrome for instance, where font sizes in `px` remain the same whatever base font size you select in the settings.
+
+Users can still Zoom, but they would have to do it on every single website they are viewing.
+Usually, when people truly need bigger characters, they want to specify the size once for instance from the Operating System settings. They expect it to be applied everywhere, including the browser interface and every website they visit.
+
+### Using relative units for everything else
+
+Using relative font sizes is not enough because when the text gets bigger, containers should also get bigger.
+If not, then the text will overflow and may become unreadable.
+
+This is why we also used relative units for every size (`padding`, `margin`, `border`, etc.).
+
+We introduced a few exceptions for some `border` and `box-shadow` styles when it did not make sense to make them scale with the font size.
+
+### Choosing the relative unit
+
+There are several [relative units in CSS](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#relative_length_units) but in most cases it comes down to two options:
+
+* `em` which is based on the font size of the parent for typographical properties (`font-size` for instance) and the element itself for other properties (`width`, `height`, etc.).
+* `rem` which is based on the font size set on the root element (the `<html>` tag).
+
+We chose to use the `rem` unit because it felt like it was the simplest to use.
+It allowed us to use a unit that we assumed would always be computed to the same value across all our components.
+
+## The problem
+
+We faced issues with two websites that modify the root font size used by the `rem` unit.
+
+In our private admin dashboard, the root font size is specified like this:
+
+```css
+html {
+  font-size: 62.5%;
+}
+```
+
+In our Documentation Website, the root font size is specified like this:
+
+```css
+html {
+  font-size: 10px;
+}
+```
+
+The goal of this is generally to simplify the conversion from `px` to `rem`.
+This way, developers only have to divide the size in pixels by 10 to get the corresponding `rem` size as explained in [The Surprising Truth about Pixels and Accessibility by Josh W Comeau](https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/#the-625-trick).
+
+For instance, `16px` equals to `1.6rem`. 
+
+The issue is that our components were built assuming that `16px` equals to `1rem` because it's the standard.
+This means our components looked a lot smaller on these two websites.
+
+One of the key principles of our Web Components is that they can be used in almost any context.
+Using `rem` units clearly fails this principle because they rely on a root font size they cannot control.
```

Comment:
This is just a suggestion to clarify a few points, you can adjust anything you want ;-)

```suggestion
While trying to use our components, we faced issues with two of our websites that modify the root font size used by the `rem` unit.

In our private admin dashboard, the root font size is specified like this:

```css
html {
  font-size: 62.5%;
}
```

It's not a bad practice as it still respect the user's settings but it completely breaks any third party CSS based on rems.

In our documentation website, the root font size is specified like this:

```css
html {
  font-size: 10px;
}
```

This is clearly a bad practice as it completely ignores the user's settings.
Sadly, this practice is not that uncommon on the web.

In both cases, the goal is generally to simplify the conversion from `px` to `rem`.
This way, developers only have to divide the size in pixels by 10 to get the corresponding `rem` size as explained in [The Surprising Truth about Pixels and Accessibility by Josh W Comeau](https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/#the-625-trick).

For instance, `16px` equals to `1.6rem`. 

The issue is that our components were built assuming that `1rem` would either be equal to `16px`, the default in most browsers or a value set by the user.
In these two situations, `1rem` is equal to `10px` so our components looked a lot smaller on these two websites.

One of the key principles of our Web Components is that they can be used in almost any context.
Using `rem` units clearly fails this principle because they rely on a root font size they cannot control.
```